src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.js

```diff
@@ -0,0 +1,17 @@
+goog.module('optional.export.object.property');
+goog.module.declareLegacyNamespace();
+
+/**
+ * @return {number|undefined}
+ */
+function getNumberOrUndefined() {}
```

Comment:
fn is not exported thus not seen in the output .d.ts. I am guessing you needed it to get around closure's type inference, but explain that in a comment (use //!! to skip emit on it).

Reply:
Yep.